package_builder/build_libgrib_api.sh

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env bash
+
+exitError()
+{
+    echo "ERROR $1: $3" 1>&2
+    echo "ERROR     LOCATION=$0" 1>&2
+    echo "ERROR     LINE=$2" 1>&2
+    exit $1
+}
+
+TEMP=$@
+eval set -- "$TEMP --"
+fwd_args=""
+resources_repo=git@github.com:MeteoSwiss-APN/libgrib-api-cosmo-resources.git
+compiler_target="all"
+while true; do
+    case "$1" in
+        --dir|-d) package_basedir=$2; shift 2;;
+        --idir|-i) install_dir=$2; shift 2;;
+        --local|-l) install_local="yes"; shift;;
+        --compiler|-c) compiler_target=$2; shift 2;;
+        --jasper_dir|-j) jasper_dir=$2; shift 2;;
+        --resources_version|-r) resources_version=$2; shift 2;;
+        --resources_repo) resources_repo=$2; shift 2;;
+        --thread_safe|-n) thread_safe=yes; shift 2;;
+        -- ) shift; break ;;
+        * ) fwd_args="$fwd_args $1"; shift ;;
+    esac
+done
+
+if [[ "${help_enabled}" == "yes" ]]; then
+    echo "Available Options for libgrib:"
+    echo "* --local             |-l {install locally}         Default=No"
+    echo "* --compiler          |-c {compiler}                Default=all"
+    echo "* --jasper_dir        |-j {jasper installation dir} Default=install_path/libjasper"
+    echo "* --resources_version |-r {resources version}       Required"
+    echo "* --resources_repo    |-r {resources repository}    Default=git@github.com:MeteoSwiss-APN/libgrib-api-cosmo-resources.git"
+    echo "* --thread_safe       |-n {thread_safe mode}        Default=False"
+    exit 0
+fi
+
+if [[ -z ${package_basedir} ]]; then
+    exitError 4201 ${LINENO} "package basedir has to be specified"
+fi
+if [[ -z ${install_dir} ]]; then
+    exitError 4202 ${LINENO} "package install dir has to be specified"
+fi
+if [[ -z ${resources_version} ]]; then
+    exitError 4203 ${LINENO} "resources_version has to be specified (coupling to libgrib)"
+fi
+
+# Setup
+echo $@
+base_path=$PWD
```

Comment:
Actually, this base path is the current directory. Which can be different from the package dir.